docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,547 @@
+Management Systems in general.
+
+************
+Introduction
+************
+
+The core mission of edX is to expand access to education for everyone. We expect
+any user interfaces that are developed for the Open edX platform to be usable by
+everyone, regardless of any physical limitations that they might have. The Open
+edX platform is used every day by people who might not be able to see or hear,
+or who might not be able to use traditional modes of computer interaction such
+as the mouse or keyboard. 
+
+Understanding a few core concepts about how people with disabilities use the web
+and web applications should give you more context for applying the guidance in
+this document.
+
+* People without vision, as well as those with neurological conditions that
+  prevent precise hand-eye coordination, cannot use a mouse.
+
+* People with mobility impairments might use custom keyboards or keyboard
+  emulators.
+
+* Some people rely on speech to interact with web applications and need to be
+  able to address interactive elements.
+
+* People who cannot hear or hear well require visible text captions, or an
+  equivalent visual cue, for any audible content.
+
+* Many people with disabilities use "`assistive technologies
+  <http://www.w3.org/TR/WCAG20/#atdef>`_" (such as screen readers) to interact
+  with their computers, web browsers, and web applications. Users might customize
+  their operating system, browser or web page display properties to make web
+  applications easier for them to use. For example, they might increase the font
+  size in their displays, reverse or increase contrast, or remove images.
+
+* To allow assistive technologies to provide the maximum information to users,
+  anything in a user interface that is conveyed visually (such as the element
+  with focus, an element's role, its current state, and other properties) must
+  also be conveyed programmatically. This information is often included when you
+  use native HTML5 elements.
+
+Keep these core concepts in mind when you develop user interfaces, so that they
+can truly be used by everyone. More information is available from the W3C's Web
+Accessibility Initiative's `How People with Disabilities Use the Web: Overview
+<http://www.w3.org/WAI/intro/people-use-web/Overview.html>`_.
+
+
+****************************
+Accessibility Best Practices
+****************************
+
+The following sections cover some best practices and tips to keep in mind as you
+develop user interfaces that are WCAG 2.0 compliant.
+
+* :ref:`Use semantic markup`
+* :ref:`Avoid using CSS to add content`
+* :ref:`Include title attributes for all iframe elements`
+* :ref:`Make sure form elements have labels`
+* :ref:`Include labels for links and interactive controls`
+* :ref:`Use WAI ARIA to create accessible widgets`
+* :ref:`Manage focus for popups`
+* :ref:`Inform users when content changes dynamically`
+* :ref:`Hide or expose content to targeted audiences`
+* :ref:`Make images accessible`
```

Comment:
I think that Making images accessible should be closer to the top, probably the second item following semantic markup.   It's the most problematic area of accessibility, yet its the easiest to understand.